frontend/public/components/resource-quota.jsx

```diff
@@ -72,11 +72,7 @@ const getResourceUsage = (quota, resourceType) => {
   };
 };
 
-const tableColumnClasses = [
-  classNames('col-md-5', 'col-xs-6'),
-  classNames('col-md-7', 'col-xs-6'),
-  Kebab.columnClass,
-];
+const tableColumnClasses = ['pf-u-w-42-on-lg', 'pf-u-w-58-on-lg', Kebab.columnClass];
```

Comment:
above change works well for `All Projects`, but not if you are in a namespace

![image](https://user-images.githubusercontent.com/9964343/122555831-68ddd500-d058-11eb-903b-88ddad9ab738.png)

Reply:
yes, I changed accordingly

![Screenshot from 2021-06-21 16-31-51](https://user-images.githubusercontent.com/24852534/122752559-be132400-d2ae-11eb-9e78-1914d4d6f43d.png)
![Screenshot from 2021-06-21 16-32-02](https://user-images.githubusercontent.com/24852534/122752567-bf445100-d2ae-11eb-9d3d-b4c2006c13a5.png)